src/pages/GardenAdmin/GardenAdmin.tsx

```diff
@@ -16,24 +19,33 @@ const GardensAdmin = (): JSX.Element => {
     withCredentials: authEnabled,
   })
 
+  const [requestStatus, setRequestStatus] = useState<SnackbarState | undefined>(
+    undefined,
+  )
+
   useEffect(() => {
     if (data && !error) {
       setGardens(data)
     }
   }, [data, error])
 
   return (
-    <Box>
+    <>
+      <CreateGarden />
+      <Box style={{ float: 'right' }}>
+        <GardenSyncButton gardenName={''} setSyncStatus={setRequestStatus} />
```

Comment:
This should probably have more logic than just producing a snackbar message. Please open a ticket to refactor this such that the passed in function will update the garden list on both success and failure (and also continue to produce the snackbar).

Reply:
Issue #199